Readme.md

```diff
@@ -0,0 +1,12 @@
+package pretty
+
+[![GoDoc](https://pkg.go.dev/badge/github.com/kr/pretty)](https://pkg.go.dev/github.com/kr/pretty)
+[![Go Report Card](https://goreportcard.com/badge/github.com/kr/pretty)](https://goreportcard.com/report/github.com/kr/pretty)
```

Comment:
It looks like this report card site uses cyclomatic complexity as a component in its score. I think CC can be a useful thing to look at, but I don't think it's reasonable to set a fixed CC threshold below which the code is considered "good" and above which it is "bad". What do you think about removing this badge?

Reply:
The badges are the most common in the gomod community on github. This is just a shortcut for people. They can make the website do the scan that you know it or not. So, the question is just up to you ;)